packages/amplify_core/ios/Classes/Support/ErrorMessages.swift

```diff
@@ -31,4 +31,13 @@ public struct ErrorMessages {
 
     public static let missingExceptionMessage: String =
         "An unknown exception has happened. " + createGithubIssueString
+
+
+    public static let unexpectedExceptionMessage: String =
+        "An unexpected exception has occurred."
```

Comment:
Probably should append `createGithubIssueString` here as well? What's the guidance between using unknown vs unexpected exception?  I'd think all unknown exceptions are unexpected?

Reply:
Agreed - this is a distinction without a difference. I've started using`missingExceptionMessage` in both Android and iOS.